tcms/testplans/templates/testplans/mutable.html

```diff
@@ -4,16 +4,19 @@
 
 {% block head %}
     {{ form.media }}
+    {% block title %}
+        {% if test_plan %}
+            {% trans "Edit TestPlan" %}
+        {% else %}
+            {% trans "Create new TestPlan" %}
+        {% endif %}
+    {% endblock %}
 {% endblock %}
 
-<head>
-    <title>{% block title %}{% trans "Create new TestPlan" %}{% endblock %}</title>
-</head>
-
 {% block contents %}
 
     <div class="container-fluid container-cards-pf">
-        <form class="form-horizontal" action="{% url 'plans-new' %}" method="post">
+        <form class="form-horizontal" action="{% if test_plan %} {% url 'plan-edit' test_plan.plan_id %} {% else %}{% url 'plans-new' %} {% endif %}" method="post">
```

Comment:
Drop the spaces around curly braces! You have the habit to put them everywhere.

In this case the rendered HTML includes spaces around the URL and while Firefox manages to deal with that it is not valid HTML and other browsers (or other versions of the same browser) may break.